Common/Configurations/EnvironmentDescription.cs

```diff
@@ -54,11 +54,6 @@ void Dispose(bool disposing)
             }
         }
 
-        public bool SettingExists(string settingName)
```

Comment:
it's mostly used from ConfigurationProvider which uses CloudConfigurationManager SDK to store/access RM configs. I'll keep them in case they're used in the future...